Example/ios/Podfile.lock

```diff
@@ -1449,4 +1449,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: fca071226b19ce84770ff3ba64fd881fcb93dead
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.15.2
```

Comment:
👀

Reply:
@Latropos Apparently, .2 fixed the 1.15.1 issues